pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -68,6 +68,7 @@ const TagNameSubnetPublicELB = "kubernetes.io/role/elb"
 // Currently we accept only the value "0.0.0.0/0" - other values are an error.
 // This lets us define more advanced semantics in future.
 const ServiceAnnotationLoadBalancerInternal = "service.beta.kubernetes.io/aws-load-balancer-internal"
+const ServiceAnnotationLoadBalancerName = "service.beta.kubernetes.io/aws-load-balancer-name"
```

Comment:
I would prefer to have the alpha/beta status in the external docs.

Is there some reason you put aws into the prefix @thockin?

My preference would be `service.kubernetes.io/load-balancer-name` or `service.kubernetes.io/aws-load-balancer-name`.  The former if we think think might be useful on multiple clouds.